src/fu-util.c

```diff
@@ -3672,10 +3672,12 @@ fu_util_modify_config(FuUtilPrivate *priv, gchar **values, GError **error)
 					   _("Restart the daemon to make the change effective?")))
 			return TRUE;
 	}
-#ifdef HAVE_SYSTEMD
-	if (!fu_systemd_unit_stop(fu_util_get_systemd_unit(), error))
+
+	if (!fu_util_quit(priv, NULL, error))
 		return FALSE;
-#endif
+	if (!fwupd_client_connect(priv->client, priv->cancellable, error))
```

Comment:
does this work, or do you get to the:

	/* nothing to do */
	if (priv->proxy != NULL) {
		g_task_return_boolean(task, TRUE);
		return;
	}

case?

Reply:
Ah... it's hitting that case, that explains why it returned so quickly.  An alternative is to change the message and just "Quit" the daemon.

Reply:
Pushed a fix.